keps/sig-node/3386-kubelet-evented-pleg/README.md

```diff
@@ -472,12 +524,11 @@ and creating new ones, as well as about cluster-level services (e.g. DNS):
 -->
 - CRI Runtime
   - CRI runtimes that are capable of emitting CRI events must be installed and running.
-    - Impact of its outage on the feature: Kubelet will detect the outage and fall back on the current default relisting period to make sure the pod statuses are updated in time.
+    - Impact of its outage on the feature: Kubelet will detect the outage and fall back on the `Generic PLEG` with the default relisting period to make sure the pod statuses are updated correctly.
```

Comment:
Just to be clear on this, if kubelet detects that the CRI runtime does not support `GetContainerEvents` (i.e. getting `codes.Unimplemented`), it will fall back to generic PLEG with 1s interval?

Reply:
Yes, that's correct.

Reply:
Is this going to be tested during startup?  How will someone know whether the kubelet is running against a cri that doesn’t support evented pleg?

Reply:
During the start up of the Evented PLEG, any error returned while establishing the grpc connection with the runtime is captured. This is how it is handled in the current implemention, https://github.com/harche/kubernetes/blob/09160203eeba6e34bc39c384873be0d9e98eb931/pkg/kubelet/pleg/evented.go#L180